cmc-domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/offers/Settlement.java

```diff
@@ -33,11 +34,27 @@ public void reject(MadeBy party) {
     @JsonIgnore
     PartyStatement getLastStatement() {
         if (partyStatements.isEmpty()) {
-            throw new IllegalSettlementStatementException("No statements have yet been made during that settlement");
+            throw new IllegalSettlementStatementException(NO_STATEMENTS_MADE);
         }
         return partyStatements.get(partyStatements.size() - 1);
     }
 
+    @JsonIgnore
+    public PartyStatement getLastOfferStatement() {
+        if (partyStatements.isEmpty()) {
+            throw new IllegalSettlementStatementException(NO_STATEMENTS_MADE);
+        }
+
+        List<PartyStatement> tmpList = new ArrayList<>(partyStatements);
+        Collections.reverse(tmpList);
+
+        return tmpList.stream()
+            .filter((partyStatement -> partyStatement.getType() == StatementType.OFFER))
+            .findFirst()
```

Comment:
Maybe we could sort it here by using .sorted(Collections.reverseOrder()) and then take first element. That could help us avoid using nasty Collections.reverse().

Reply:
That only works if there’s a comparator we rely on natural ordering. I tried this first

Reply:
That only works if there’s a comparator we rely on natural ordering. I tried this first